community/releases/release-manager-runbook/index.md

```diff
@@ -119,9 +146,11 @@ this message, including a link to the correct section of the changelog gist:
 [#dev](https://spinnakerteam.slack.com/messages/dev/) with a message that the
 release candidate is now validated and can be tested by running:
 
-    ```
-    hal config version edit --version ${RELEASE_BRANCH}-latest-validated
-    ```
+    > You are now welcome to test out the new ${RELEASE} by doing
```

Comment:
On the one hand, this is better because it shows the whole message. But on the other hand, the intro sentence makes less sense because it tries to introduce just the suggested command, not the informational message. (That is, you wouldn't "run" `You are welcome to test...`)